### ISSUE TITLE
How to build RPMs

### ISSUE THREAD

**Issue by tdudgeon** [natefoo/slurm-drmaa#25]
Is it possible to build RPMs from the source build?
Instructions?

**Comment by natefoo** [comment]
Yes, I've provided an [RPM spec file](https://github.com/natefoo/slurm-drmaa/blob/master/slurm-drmaa.spec). Using that, follow the standard procedure for building using `rpmbuild`. An rpmbuild guide can be found [here](https://rpm-packaging-guide.github.io/) if you're new to it.
